setup.py

```diff
@@ -15,6 +15,24 @@
 import versioneer
 
 
+def install_custom_ops() -> bool:
+    """Determine if we should install the custom ops.
+
+    If it is not a linux machine and doesn't have at least nvidia-smi we skip it. Can
+    still use the reference platform if needed in such cases.
+    """
+    if os.environ.get("SKIP_CUSTOM_OPS"):
+        return False
+    if "linux" not in sys.platform:
+        return False
+    try:
+        subprocess.check_call(["nvidia-smi"])
```

Comment:
nit: should this check for `nvcc`? This seems more accurate in the edge case where the toolkit but not runtime environment is installed.

Reply:
Good call, fixed in https://github.com/proteneer/timemachine/pull/944/commits/7d372f6d16783b07cb916a1f350e6efb4a3b32a3